job-service-db/src/main/resources/procedures/insertDeleteLog.sql

```diff
@@ -0,0 +1,37 @@
+--
+-- Copyright 2016-2021 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *  Name: internal_insert_delete_log
+ *
+ *  Description:
+ *  This is used to insert a record into delete log table.
+ */
+DROP FUNCTION IF EXISTS internal_insert_delete_log(
+    task_table_name VARCHAR
+);
+
+CREATE OR REPLACE FUNCTION internal_insert_delete_log(
+    task_table_name VARCHAR
+)
+RETURNS VOID
+LANGUAGE plpgsql
```

Comment:
I know `VOLATILE` is the default but I'd prefer if we were always explicit w.r.t. the volatility category (to show that we've thought about it).
```suggestion
LANGUAGE plpgsql VOLATILE
```